Add 16 Heads As A Test Notebook

ArthurConmy opened the pull request.

rhaps0dy commented on acdc/acdc_utils.py:
```diff
@@ -181,9 +183,18 @@ def kl_divergence(
         denom = mask_repeat_candidates.long().sum()
         kl_div = kl_div * mask_repeat_candidates
-        answer = kl_div.sum() / denom
+        if return_one_element:
```
Why is this needed? Seems like a thing that makes mistakes easy?

rhaps0dy commented on acdc/acdc_utils.py:
```diff
@@ -181,9 +183,18 @@ def kl_divergence(
+        else:
+            answer = kl_div
+
+    if not return_tensor:
```
Seems easier to just call `.item()` on the output of `kl_divergence` if it's needed

ArthurConmy commented on acdc/TLACDCExperiment.py:
```diff
@@ -132,7 +132,7 @@ def __init__(
             )
 
-        self.metric = metric
+        self.metric = lambda x: metric(x).item()
```
New way of using both tensors and floats through the code

ArthurConmy commented:
@rhaps0dy agreed that passing round that flag is annoying! 

I removed it and added the new way of using both tensors and floats, how does that sound?

rhaps0dy commented:
Sounds great, hit that merge button!
